dsn.go

```diff
@@ -13,21 +13,64 @@
 
 package goalisa
 
-// Config is the deserialization of connect string
+import (
+	"fmt"
+	"net/url"
+	"regexp"
+)
+
+var (
+	reDSN = regexp.MustCompile(`^([a-zA-Z0-9_-]+):([=a-zA-Z0-9_-]+)@([:a-zA-Z0-9/_.-]+)\?([^/]+)$`)
+)
+
+// Config is the deserialization of connect string, the connection string should of format:
+// pop_access_id:pop_access_key@pop_url?alisa_access_id=..&alisa_access_key=..&alisa_project=..&env=..
 type Config struct {
-	AccessID  string
-	AccessKey string
-	Endpoint  string
+	// POP config
+	POPAccessID  string
+	POPAccessKey string
+	POPURL       string
+	// Alisa config
+	AlisaAccessID  string
+	AlisaAccessKey string
+	AlisaProject   string
```

Comment:
The `ENV` already carries the `Alisa*` properties.

Reply:
I see. So we don't need the Alisa* field?

Reply:
No, we don't need these fields.